doc/classes/String.xml

```diff
@@ -646,6 +646,12 @@
 				Returns the similarity index of the text compared to this string. 1 means totally similar and 0 means totally dissimilar.
 			</description>
 		</method>
+		<method name="simplify_path" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns a simplified canonical path.
```

Comment:
This should probably explain what a simplified canonical path is?

Reply:
An explanation is being added on #52523.